docs/CONFIG-PROPERTIES.md

```diff
@@ -36,6 +36,7 @@
 | force.fallback.counter | integer | 0 | forces fallback to other image if counter is changed |
 | newlog.allow.fastupload | boolean | false | allow faster upload gzip logfiles to controller |
 | memory.apps.ignore.check | boolean | false | Ignore memory usage check for Apps|
+| newlog.gzipfiles.ondisk.maxmegabytes | integer in Mbytes | 0xFFFFFFFF | the quota for keepig newlog gzip files on device |
```

Comment:
What does the default MAXINT-1 mean? What would zero mean?

Reply:
changed to 2048 Mbytes